src/search/views/Search.tsx

```diff
@@ -338,28 +338,26 @@ const Search: FunctionComponent<SearchProps & RouteComponentProps> = ({
 									</DropdownButton>
 									<DropdownContent>
 										<Fragment>
-											<a
+											<Button
+												type="link"
 												className="c-menu__item"
+												label="Kopieer vaste link naar deze zoekopdracht"
 												onClick={() => {
```

Comment:
Achteraf gezien zou ik deze functie proberen uit de template te halen aangezien het toch al de volle 5 lijnen inneemt. Zou readability wel improven om de ietwat grotere methods te seperaten van de templates.

Reply:
fixed in future PR